src/blocks/common.blocks/block-example/block-example.bemtree

```diff
@@ -71,6 +73,7 @@ block('block-example')(
                     {
                         elem: 'blank',
                         url: url,
+                        isIE8: this._isIE8,
```

Comment:
why not to generate `target` instead of `isIE8 ` right here?

```js
{
    elem: 'blank',
    url: url,
    target: req.headers['user-agent'].indexOf('MSIE 8.0') < 0 ? '_blank' : undefined
}
```